styles.css

```diff
@@ -1242,4 +1353,245 @@ label {
+    transform: scale(1.05);
+    background-image: url("images/1st.png");
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .details1:hover {
+    transform: scale(1.05);
+    background-image: url("images/1st.png");
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .details.details2 {
+    margin: 12px;
+    align-items: center;
+    transition: all 0.5s ease-in-out;
+  }
+
+  .details1.details1 {
+    margin: 12px;
+    align-items: center;
+    transition: all 0.5s ease-in-out;
+  }
+
+  .details.details {
+    margin: 12px;
+    align-items: center;
+    transition: all 0.5s ease-in-out;
+  }
+
+  .details.details1 .hov {
+    display: block;
+    width: 100%;
+  }
+
+  .details.details2.details1.details.details1:hover {
+    /* transform: scale(1.08); */
+    background-image: url("images/1st.png");
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .details:hover {
+    /* transform: scale(1.08); */
+    background-image: url("images/1st.png");
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .details2.details2:hover .hov {
+    display: block;
+    width: 100%;
+    margin-top: 24rem;
+  }
+
+  .details1.details1:hover .hov {
+    display: block;
+    width: 100%;
+    margin-top: 20rem;
+  }
+
+  .mobile-heading {
+    display: none;
+  }
+
+  .details.details:hover .hov {
+    display: block;
+    width: 100%;
+    margin-top: 20rem;
+  }
+
+  .details2.details2:hover h2 {
+    display: none;
+  }
+
+  .details1.details1:hover h2 {
+    display: none;
+  }
+
+  .details.details:hover h2 {
+    display: none;
+  }
+
+  .details2.details2:hover p {
+    display: none;
+  }
+
+  .details.details:hover p {
+    display: none;
+  }
+
+  .details1.details1:hover p {
+    display: none;
+  }
+
+  .details2.details2:hover ul li {
+    display: none;
+  }
+
+  .details.details:hover ul li {
+    display: none;
+  }
+
+  .details1.details1:hover ul li {
+    display: none;
+  }
+
+  .recent-para2 {
+    display: block;
+    width: 95%;
+    height: 300px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 30px;
+    color: #344563;
+    margin-top: 22px;
+  }
+
+  .recent-para1 {
+    max-width: 685px;
+    width: 747px;
+    height: 226px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 30px;
+    color: #344563;
+  }
+
+  .recent-para5 {
+    display: none;
+  }
+
+  .recent-pic {
+    align-items: center;
+    width: 100%;
+  }
+
+  .destop5 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .btn {
+    display: flex;
+    justify-content: center;
+    gap: 2rem;
+    margin: 14px 16px;
+  }
+}
+
+/* --------------------popup------------ */
+
+.popUp {
+  display: block;
+  position: fixed;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  background-color: white;
+  z-index: 1000;
+  width: 80%;
+}
```

Comment:
- [ ] Nice work implementing the popup windows :clap: :sunglasses: However popup's image are not showing for both mobile and desktop version, kindly check it and make sure you follow the Figma design.
### Your design :point_down: 
![Screenshot from 2023-04-22 10-10-17](https://user-images.githubusercontent.com/91142432/233779091-344586aa-8569-465c-b86d-3a5a7352e291.png)

### Figma's Design :point_down: 
![Screenshot from 2023-04-18 17-52-27](https://user-images.githubusercontent.com/91142432/233779124-a23f20a6-3ad0-41e0-b40e-c8171e46ef9b.png)

Reply:
resolved thanks